gocourse03/mouse_city.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+const RodentRat rodentType = "Rat"
+const SectorCenter Sector = "Sector Center"
+
+type (
+	Sector     string
+	rodentType string
+	FromTo     [2]Sector
+
+	DailyMovements struct {
+		RodentMovement []Movement
+	}
```

Comment:
Спробуймо оголосити `DailyMovements` ось так
```suggestion
	DailyMovements []Movement
```
та підлаштувати решту коду під це.

Має працювати.

Reply:
Ось як тут `type Persons []Person`
https://go.dev/play/p/361NYZpFqvf

Reply:
Виправила